SPECS/enchant/enchant.spec

```diff
@@ -1,41 +1,42 @@
+Summary:        An Enchanting Spell Checking Library
+Name:           enchant
+Version:        1.6.0
+Release:        28%{?dist}
+License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Summary: An Enchanting Spell Checking Library
-Name: enchant
-Version: 1.6.0
-Release: 27%{?dist}
-License: LGPLv2+
-Source: http://www.abisource.com/downloads/enchant/%{version}/enchant-%{version}.tar.gz
-URL: http://www.abisource.com/
+URL:            https://www.abisource.com/
+Source:         https://src.fedoraproject.org/repo/pkgs/enchant/enchant-1.6.0.tar.gz/
+#Source:         http://www.abisource.com/downloads/enchant/%{version}/enchant-%{version}.tar.gz
```

Comment:
Why two different URLs? Please just use the one for our source, otherwise it's confusing.

Reply:
I've been testing different URLs for enchant/malaga since both are failing remotely while building locally. It seems to be something about the way validate-cg-manifest.sh greps the wget results.